extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -231,5 +186,95 @@ describes.realWin(
         ).to.have.been.calledOnce;
       });
     });
+
+    describe('amp-story-shopping-config validation', () => {
+      it('should fail config validation because a required config value is missing', async () => {
+        const invalidConfig = JSON.parse(JSON.stringify(defaultInlineConfig));
```

Comment:
Why `JSON.parse(JSON.stringify())`?

If you're trying to create a copy, maybe name a utility function to make this clearer:

```js
const copyObject = (object) => JSON.parse(JSON.stringify(object));
```